### DISCUSSION TITLE
Any approach to curate tsvs locally tagged?

### DISCUSSION THREAD

**Discussion by egumasa** [inception-project/inception#3455] [Q&A] [answered]
Hello,

I love this tool and its predecessor, webanno, as a free annotation tool. As I use the functionality locally, because I do not have servers to host the web app, I wonder if there is any get-around to curate separate tsvs which are locally tagged on different computers (under two copied projects). 

I tried to unzip the project, put two separate tsvs under the folders, and compress the project again, but this get-around does not seem to work. Is there any way to make this possible?

Thank you so much again for the great tool!

**Comment by reckart** [comment, marked as answer]
It is not quite as easy as that. First, when importing a project, INCEpTION would only use the `.ser` files. Second, if you just copy the `.ser` files there, it still won't work because INCEpTION does not know about them. You would also have to update the `.json` file that is at the root of the project export to reference these new files. I would not recommended to do that though for this purpose.

What you could do is, you could enable the remote API and then use it to upload externally created annotations into specific users in your local annotation projects. Normally, the remote API us for use with scripting languages like via the Python [pycaprio](https://pycaprio.readthedocs.io/en/latest/) library - but INCEpTION also includes a minimal Swagger UI which allows to use the remote API directly from a browser.

See: https://inception-project.github.io/releases/25.1/docs/admin-guide.html#sect_remote_api

**Reply by egumasa** [reply]
Thank you very much for pointing me in the right direction. I will learn these file formats and API and come back to this post if I have any difficulties. Thank you again for developing and maintaining the great, user-friendly annotation tool!!
